services/listings/package.json

```diff
@@ -14,24 +14,58 @@
     "@types/supertest": "^2.0.8",
     "babel-jest": "^25.2.3",
     "concurrently": "^5.1.0",
+    "eslint": "^7.0.0",
     "jest": "^25.2.3",
     "supertest": "^4.0.2",
-    "ts-node-dev": "^1.0.0-pre.44"
+    "ts-node-dev": "^1.0.0-pre.44",
+    "tsconfig-paths": "^3.9.0"
   },
   "dependencies": {
     "@bloom-housing/core": "^0.1.0",
     "@koa/cors": "^3.0.0",
+    "@storybook/react": "^5.3.14",
```

Comment:
We need storybook for Ts.ED?

Reply:
Good catch. I am not sure how it got there in the first place.